src/apps/categories/dataAccess/categoryModel.js

```diff
@@ -1,13 +1,13 @@
 import { applyErrorHandlingMiddleware } from '../../../errors/utils/dbErrorHandling.js';
 import mongoose, { Schema } from 'mongoose';
 
-//TODO: Indexing
+//TODO: Indexing, commented required should be handled by backend and not received from request
 const categorySchema = new Schema(
   {
     title: { type: String, required: true, unique: true },
-    managedBy: { type: String, required: true },
+    managedBy: { type: String }, //required: true
```

Comment:
this field is not required any more when u creating category?

Reply:
this field will be filled using the JWT token, this should not be sent by the user or in the body request